ui-v2/app/models/service.js

```diff
@@ -21,6 +21,7 @@ export default Model.extend({
   EnableTagOverride: attr('boolean'),
   CreateIndex: attr('number'),
   ModifyIndex: attr('number'),
+  // TODO: These should be typed
   ChecksPassing: attr(),
   ChecksCritical: attr(),
   ChecksWarning: attr(),
```

Comment:
I'm trying to avoid making any non-ACL changes in this PR, I'll probably hit this in my spring clean before final merge.